decidim-elections/app/views/decidim/elections/admin/trustees/edit.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= decidim_form_for(@form, html: { class: "form edit_trustee" }) do |f| %>
```

Comment:
As commented in the `TrusteesController` I think this view is not used any more.